File: BECore/app/model/ProjectPlanExpense.js
/// <summary>
/// Model EXTJS ProjectPlanExpense.
/// </summary>
Ext.define('QLDT.model.ProjectPlanExpense', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectPlanExpenseID',
    idType: 'Guid',
    fields: [	
		{ name:"ProjectID", type:"string" },
		{ name:"Number", type:"string" },
		{ name:"Date", type:"date" },
		{ name:"Amount", type:"float" },
		{ name:"Inactive", type:"boolean" },
		{ name:"SortOrder", type:"int" },
		{ name:"CreatedDate", type:"date" },
		{ name:"ModifiedDate", type:"date" },
		{ name:"IPAddress", type:"string" },
		{ name:"ModifiedBy", type:"string" },
		{ name: "CreatedBy", type: "string" },
        { name: 'IdFiles', type: 'string' }
     ]
});

File: MT.Model/Business/ProjectPlanExpense.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
	#region ProjectPlanExpense:EntityBase
	/// <summary>
	/// This object represents the properties and methods of a ProjectPlanExpense:EntityBase.
	/// </summary>
	public class ProjectPlanExpense:EntityBase
	{
        [Key]
		public Guid ProjectPlanExpenseID {get;set;}
		public Guid ProjectID{get;set;}
		public string Number{get;set;}
		public DateTime Date{get;set;}
		public decimal Amount{get;set;}
		public bool Inactive{get;set;}
		public int SortOrder{get;set;}
        /// <summary>
        /// Danh sách file đính kèm
        /// </summary>
        /// Create by: truongnm:11.02.2018
        public List<ProjectPlanExpense_AttachDetail> Details { get; set; }

        public string IdFiles { get; set; }
    }
	#endregion
}

